spec/catalog/api/minion/approval_spec.rb

```diff
@@ -15,19 +15,30 @@
     let(:event) { "request_started" }
     let(:config) do
       test_config = ::CatalogApiClient.configure
-      test_config.host = "localhost:3000"
+      test_config.host = "localhost.com:3000"
```

Comment:
Not that it matters, but any reason you added the `.com`? I think if you just used `localhost:3000` everywhere it should still work.